### ISSUE TITLE
CodeAction with groups?

### ISSUE THREAD

**Issue by simrat39** [simrat39/rust-tools.nvim#80]
![image](https://user-images.githubusercontent.com/37233846/138173655-b215ab2b-5983-4ba3-9901-c1387ff83fdf.png)
 hmmmmmmmmmmmmmmmm

**Comment by sharksforarms** [comment]
To support this: https://github.com/rust-analyzer/rust-analyzer/pull/10539#issuecomment-944675573 ?

**Comment by simrat39** [comment]
> To support this: [rust-analyzer/rust-analyzer#10539 (comment)](https://github.com/rust-analyzer/rust-analyzer/pull/10539#issuecomment-944675573) ?

Yes, also pushed to master :)

**Comment by simrat39** [comment]
![image](https://user-images.githubusercontent.com/37233846/138191613-d962680c-5b3a-4333-a428-fe1a19469a7e.png)

vs

![image](https://user-images.githubusercontent.com/37233846/138191649-0ecf9109-4c96-46e5-81e2-0ec1e8dba05a.png)

Also that is one long boi, def need to have max length

**Comment by sharksforarms** [comment]
Hmm might be a better candidate for telescope if >some length

**Comment by simrat39** [comment]
> Hmm might be a better candidate for telescope if >some length

Yes prolly should add that as an option, but for now / and ? should suffice
